Cargo.lock

```diff
@@ -1,6 +1,6 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
+version = 4
```

Comment:
This is why the build is failing. Best guess: you used Rust 1.85 while updating, rather than the Rust version from the Rust toolchain file. Which is odd: if you’re using rustup, that *should* have used the local version automatically? In any case, should be a quick fix.